doc/documentation_workflow.rst

```diff
@@ -0,0 +1,152 @@
+Documentation workflow
+######################
+
+What you need to know
++++++++++++++++++++++
+
+We use **Sphinx** to generate documentation and **Read the Docs** to publish it.
+
+The documentation for NEST is contained in the ``docs`` directory within the `NEST source code repository <https://github.com/nest/nest-simulator>`_ on GitHub.
+
+Using **GitHub** allows the NEST simulator documentation to live alongside its code. Our developers benefit from smooth version control, with each release of NEST having its own documentation.
```

Comment:
Isn't it Git that allows us to keep the versions under control, while GitHub is just the service we are currently using to run a Git server for us?

Reply:
Yes

Reply:
I agree and have clarified that now.

Reply:
I agree, we should clarify that. How about:

``We use Sphinx to generate documentation and Read the Docs to publish it.``

``The NEST simulator documentation lives alongside its code. It is contained in the docs directory within the NEST source code repository on GitHub.``

``We work with GitHub as the web-based hosting service for Git. Git allows us to keep our versions under control, with each release of NEST having its own documentation.``

Reply:
Ok for me.

Reply:
Done.